ceph_deploy/gatherkeys.py

```diff
@@ -1,84 +1,262 @@
 import os.path
 import logging
+import json
+import tempfile
+import shutil
+import time
 
 from ceph_deploy import hosts, exc
 from ceph_deploy.cliutil import priority
+from ceph_deploy.lib import remoto
 
 
 LOG = logging.getLogger(__name__)
 
 
-def fetch_file(args, frompath, topath, _hosts):
-    if os.path.exists(topath):
-        LOG.debug('Have %s', topath)
-        return True
-    else:
-        for hostname in _hosts:
-            filepath = frompath.format(hostname=hostname)
-            LOG.debug('Checking %s for %s', hostname, filepath)
-            distro = hosts.get(hostname, username=args.username)
-            key = distro.conn.remote_module.get_file(filepath)
+def _keyring_equivilent(keyring_one, keyring_two):
```

Comment:
Note: Keyrings have inconsistent formatting across ceph.

The reason for this code is because I don’t want other people using this function outside the context of checking if keyring have changed.

Changing spelling.